mountpoint-s3-fs/src/fuse.rs

```diff
@@ -36,12 +38,19 @@ macro_rules! event {
 /// Handle an error in a FUSE handler. This logs the appropriate error message and then calls
 /// `reply` on the given replier with the error's corresponding errno.
 macro_rules! fuse_error {
-    ($name:literal, $reply:expr, $err:expr) => {{
+    ($name:literal, $reply:expr, $err:expr, $error_callback:expr, $request_id:expr) => {{
```

Comment:
Consider taking (references to) `S3FuseFilesystem` and `Request` directly, if it simplifies the calling code.